src/modules/launcher/Directory.Build.targets

```diff
@@ -2,12 +2,12 @@
   
    <Target Name="KillPowerLauncher" BeforeTargets="PreBuildEvent;BeforeClean">
     <Message Text="Killing process 'PowerLauncher.exe'" Importance="normal" ContinueOnError="true" />
-     <Exec Command="taskkill /F /IM PowerLauncher.exe"
+     <Exec Command="tasklist /FI &quot;IMAGENAME eq PowerLauncher.exe&quot; 2&gt;NUL | find /I /N &quot;PowerLauncher.exe&quot;&gt;NUL &#xD;&#xA; if %25ERRORLEVEL%25==0 ( taskkill /f /im PowerLauncher.exe )"
```

Comment:
Awesome.. this will prevent the taskkill from running if PowerLauncher.exe is not in the tasklist correct?

Reply:
Yes.